presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/PicPhotoCard.kt

```diff
@@ -147,43 +207,193 @@ private fun PicCroppedImageContainer(
 }
 
 @Composable
-fun PicCroppedPhoto(
+fun PicBackCardImage(
     modifier: Modifier,
-    imageUrl: String,
-    @DrawableRes frameResId: Int,
-    backgroundColor: Color = Gray0,
+    cardBackImageList: List<CardBackImage>,
+    backgroundColor: Color,
 ) {
-    PicCroppedImageContainer(
-        modifier = modifier,
-        frameResId = frameResId,
-        backgroundColor = backgroundColor,
+    LazyVerticalGrid(
+        modifier = modifier.wrapContentSize(),
```

Comment:
```suggestion
        modifier = modifier,
```
인자로 modifier 를 받고 있는데, LazyVerticalGrid 에서 `modifier.wrapContentSize()` 코드가 들어가있으면 바깥에서 아무리 modifier 를 변경하여도 wrapContentSize 로 강제될 것 같아서 혼동을 줄 수 있을것 같아!
wrapContentSize 는 사용처쪽에 책임을 넘기는건 어떨까?

Reply:
음 사실 이거를 작성하면서 나도 고민을 많이 했는뎅 wrapContentSize를 강제하기 위함이 맞긴해서..ㅠ 안그럼 Lazy가 맨 밑에까지 뻗어나가서 사진이 가운데로 정렬이 안되더라고..? 그래서 강제로 설정해둔거긴해..! 그래도 뺴두는게 나을까..? 포토 카드 안 내용물은 고정되어있는게 맞다고 생각했어서..!

Reply:
아..! 그렇구나! 그렇다면 지금 코드 그대로 유지하는게 더욱 좋을 것 같아
확인 고마워~~